tools/bridge/bridge/confirmation_sender.py

```diff
@@ -16,6 +16,8 @@
 from bridge.event_fetcher import FetcherReachedHeadEvent
 from bridge.utils import compute_transfer_hash
 
+logger = logging.getLogger(__name__)
```

Comment:
As I said, I'm ok with the change, but curious to hear your arguments for having the logger on the module instead of the instance.

Reply:
Well, I would say that the `logger = logging.getLogger(__name__)` is pretty much standard usage. You get the logger hierarchy for free, because it mirrors the python module hierarchy. There is no good reason to set it inside a class. When there is good reason, I'm fine with having it in the class (we still have it somewhere).